modules/apps/okular.nix

```diff
@@ -43,6 +43,53 @@ in with lib.types;
         default = null;
         type = nullOr bool;
       };
+
+      viewMode = lib.mkOption {
+        description = "The view mode for the pages.";
+        default = "Single";
+        type =
+          nullOr (enum [ "Single" "Facing" "FacingFirstCentered" "Summary" ]);
+      };
+
+      zoomMode = lib.mkOption {
+        description = ''
+          Specifies the default zoom mode for file which were never opened before. For those files which were opened before the previous zoom mode is applied.
+          0: "100%"
+          1: "Fit Width"
+          2: "Fit Page"
+          3: "Auto Fit"
+          '';
+        default = 1;
+        type = nullOr (enum [ 0 1 2 3 ]);
+      };
+
+      obeyDrm = lib.mkOption {
+        description =
+          "Whether Okular should obey DRM (Digital Rights Management) restrictions. DRM limitations are used to make it impossible to perform certain actions with PDF documents, such as copying content to the clipboard. Note that in some configurations of Okular, this option is not available.";
```

Comment:
Move to a multi-line description and split the lines. I.e. something like:
```nix
description = ''
  Whether Okular should obey DRM (Digital Rights Management) restrictions.
  DRM limitations are used to make it impossible to perform certain actions
  with PDF documents, such as copying content to the clipboard. Note that
  in some configurations of Okular, this option is not available.
'';
```

Reply:
Split over several lines.